### PR TITLE
Updating toolbar to support results section

### PR DESCRIPTION
**Pull request by ryelo**
image

### CODE REVIEW

File: src/PresentationalComponents/TableToolbar/TableToolbar.js [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
 
-const TableToolbar = ({ className, children, ...props }) => {
+const TableToolbar = ({ results, className, children, ...props }) => {
 
     const tableToolbarClasses = classNames(
         'ins-c-table__toolbar',
         className
     );
 
+    function generateResults(results) {
```
Comment (karelhala) [inline src/PresentationalComponents/TableToolbar/TableToolbar.js]: I might have been too fast with the approve button. Can we move this function above `TableToolbar` component? So it is not recreated everytime we re render table toolbar?

File: src/PresentationalComponents/TableToolbar/TableToolbar.test.js [hunk 2]
```diff
@@ -8,4 +8,19 @@ describe('TableToolbar component', () => {
         const wrapper = shallow(<TableToolbar>Some</TableToolbar>)
         expect(toJson(wrapper)).toMatchSnapshot();
     });
+
+    it('should render with results of 0', () => {
+        const wrapper = shallow(<TableToolbar result={ 0 }>Some</TableToolbar>)
```
Comment (karelhala) [inline src/PresentationalComponents/TableToolbar/TableToolbar.test.js]: Typo in all tests `result` -> `results`

Comment (ryelo) [reply src/PresentationalComponents/TableToolbar/TableToolbar.test.js]: 🤦‍♂️

**Pull request merged**